include/verilatedos.h

```diff
@@ -438,22 +438,22 @@ using ssize_t = uint32_t;  ///< signed size_t; returned from read()
 #if defined(__i386__) || defined(__x86_64__)
 // The vluint64_t argument is loaded with a high-performance counter for profiling
 // or 0x0 if not implemented on this platform
-#define VL_RDTSC(val) \
+#define VL_GET_CPUTICK(val) \
```

Comment:
```suggestion
#define VL_GET_CPU_TICK(val) \
```

To match VL_CPU_TICK you define elsewhere.  Similar changes below.